examples/data/hypercubes/pivot/hypercube-pivot.go

```diff
@@ -45,7 +45,7 @@ func main() {
 					Def: "Sum(Value)",
 				},
 			}},
-			Mode:                "EQ_DATA_MODE_PIVOT",
+			Mode:                "DATA_MODE_PIVOT",
```

Comment:
Why is this changed?

Reply:
According to the API specification "EQ_DATA_MODE_PIVOT" is not allowed as a Mode for a HyperDef, it should be "DATA_MODE_PIVOT".